entity-services/src/main/xdmp/entity-services/entity-services-impl.xqy

```diff
@@ -1008,7 +1010,7 @@ declare function esi:extraction-template-generate(
                                 <tde:column>
                                     { comment { "This column joins to primary key of",
                                                 $ref-type-name } }
-                                    <tde:name>{ $ref-primary-key }</tde:name>
+                                    <tde:name>{ $property-name || "_" || $ref-primary-key }</tde:name>
```

Comment:
Just FYI... I have in my tdex removed $property-name_ for the column name and it works OK.